site/src/pages/WorkspacePage/WorkspaceTopbar.tsx

```diff
@@ -201,18 +201,13 @@ export const WorkspaceTopbar: FC<WorkspaceProps> = ({
 			</div>
 
 			{!isImmutable && (
-				<div
-					css={{
-						display: "flex",
-						alignItems: "center",
-						gap: 8,
-					}}
-				>
+				<div className="flex items-center gap-4">
```

Comment:
curious: why is it better than `css` property?

Reply:
Because we are moving away from MUI and emotion to use shadcn and TailwindCSS. So we want to be consistent and only use the new stack.

Reply:
Thanks for explanation, TIL!